cmd/gomd/commands.go

```diff
@@ -57,18 +58,44 @@ func (a *app) executeCommand(command string) {
 	a.cmd.SetText("")
 }
 
-func (a *app) cmdCopy() error {
+func (a *app) getPaths() (string, string) {
```

Comment:
Great idea with this path. But the signature is at this point confusing without documentation. I would recomand to include the return variables. I would prefer `leftPath` and `rightPath` instead of `path1` and `path2` 

`func (a *app) getPaths() (leftPath, rightPath string) {`

Reply:
that's a good idea, thanks for the suggestions